Kafka reactive getting started (#880)
quarkusio/quarkus-quickstarts: kafka-quickstart/README.md

## Start the application

The application is composed of two microservices. They can be started using: 

```bash
mvn -f producer quarkus:dev
```

and

```bash
mvn -f processor quarkus:dev
```

_NOTE_: Quarkus Dev Services starts a Kafka broker for you automatically. 

Then, open your browser to `http://localhost:8080/quotes.html`.
You can send quote requests and observe received quotes.

## Anatomy

The application is composed of the following microservices:

#### Producer

* `QuoteProducer` generates uniquely identified quote requests and sends them to the Kafka topic `quote-requests`.
It also consumes the Kafka topic `quotes` and relays received messages to the browser using Server-Sent Events.
* `quotes.html` sends quote requests to the previous endpoint and updates quotes with received prices.

#### Processor

* `QuoteProcessor` consumes quote request id's from the `quote-requests` Kafka topic and responds back to the `quotes` topic with a `Quote` object containing a random price.

Interactions with Kafka is managed by MicroProfile Reactive Messaging.
Application configurations are located in `application.properties` files.

> ## Running the application in Docker

Author: We ran the dev mode in the beginning

> This project illustrates how you can interact with Apache Kafka using MicroProfile Reactive Messaging.

Reviewer: ```suggestion
This project illustrates how Quarkus application you can interact with Apache Kafka using Reactive Messaging.
```

## Start the application

The application is composed of two applications communicating through Kafka.
Interactions with Kafka is managed by MicroProfile Reactive Messaging.

They can be started in dev mode using:

```bash
mvn -f producer quarkus:dev
```

> and

Reviewer: ```suggestion
and, in another terminal:
```

```bash
mvn -f processor quarkus:dev
```

_NOTE_: Quarkus Dev Services starts a Kafka broker for you automatically. 

Then, open your browser to `http://localhost:8080/quotes.html`.
You can send quote requests and observe received quotes.

## Anatomy

The application is composed of the following components:

#### Producer

> The _producer_ application sends data to the Kafka broker.

Reviewer: ```suggestion
The _producer_ application receives requests from the user (from HTTP) and sends data to the Kafka broker.
```

Two main components compose the application:

* `QuoteProducer` generates uniquely identified quote requests and sends them to the Kafka topic `quote-requests`.
It also consumes the Kafka topic `quotes` and relays received messages to the browser using Server-Sent Events.
* `quotes.html` sends quote requests to the previous endpoint and updates quotes with received prices.

#### Processor

> The _processor_ application receives data from Kafka.

Reviewer: ```suggestion
The _processor_ application receives data from Kafka, processes them, and writes the result into the `quotes` Kafka topic.
```

> Then run with:

Author: It is a bit implicit isn't it?
